roles/distributive/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 - name: install distributive, health check packages for current role
   become: yes
   yum:
-    state: present
+    state: latest
```

Comment:
We were having that discussion in Slack the other day. That's my preferred option, I'll make it so.